tests/src/Common/CoreCLRTestLibrary/Utilities.cs

```diff
@@ -57,6 +57,8 @@ public static bool Verbose
             }
         }
 
+        public static bool IsMacOSX => (Environment.OSVersion.Platform == PlatformID.MacOSX);
+        public static bool IsUnix => (Environment.OSVersion.Platform == PlatformID.Unix);
         public static bool IsWindows => (Path.DirectorySeparatorChar == '\\');
```

Comment:
Could you update the Windows check so it is the same type of check?

Reply:
@AaronRobinsonMSFT  I changed the OS checks to use RuntimeInformation.IsOSPlatform() since Environment.OSVersion.Platform is [not reliable](https://docs.microsoft.com/en-us/dotnet/api/system.environment.osversion?view=netframework-4.7.2). It didn't work correctly for Mac.